README.md

```diff
@@ -1,43 +1,63 @@
 <h1 align="center">
   Awesome Fabric
-  <p align="center"><img src="https://awesome.re/badge-flat.svg" href="https://awesome.re" /></p>
+  <p align="center">
+    <img src="https://awesome.re/badge-flat.svg" href="https://awesome.re" />
+    <img src="https://flat.badgen.net/badge/license/CC0/blue" href="https://creativecommons.org/publicdomain/zero/1.0/" />
+  </p>
 </h1>
 
 <p align="center"><i>A curated list of awesome Fabric resources, libraries and tools - WIP and accepting contributions!</i></p>
 <p align="center"><b><a href="https://fabricmc.net/">Fabric</a> is a mod loader for the voxel game <a href="https://en.wikipedia.org/wiki/Minecraft">Minecraft</a>.</b></p>
 
 ---
 
-<h2 align="center">Contents</h2>
+<h2 align="center">Contents<br /><code>📑</code></h2>
 
 <div align="center">
-  
-| Category                        | Description                                                                    |
-| ------------------------------- | ------------------------------------------------------------------------------ |
-| [**Resource**](#resource)       | Learning resources for modding in Fabric.                                      |
-| [**Development**](#development) | Mods that help during the development of other mods.                           |
-| [**Library**](#library)         | Mods whose functionality is to be used inside other mods.                      |
-| [**Tool**](#tool)               | **External** programs that generally help with Fabric mod/modpack development. |
-
+  <table>
+    <tr>
+      <th colspan=2>Category</th> <th>Description</th>
+    </tr>
+    <tr>
+      <td>📖</td>
+      <td><a href="#resource">Resource</a></td>
+      <td>Learning resources for modding in Fabric.</td>
+    </tr>
+    <tr>
+      <td>🛠️</td>
+      <td><a href="#development️">Development</a></td>
+      <td>Mods that help during the development of other mods.</td>
+    </tr>
+    <tr>
+      <td>💾</td>
+      <td><a href="#library">Library</a></td>
+      <td>Mods whose functionality is to be used inside other mods.</td>
+    </tr>
+    <tr>
+      <td>🧰</td>
+      <td><a href="#tool">Tool</a></td>
+      <td>**External** programs that generally help with Fabric mod/modpack development.</td>
```

Comment:
Markdown formatting of `**External**` is broken this way, it should probably be `<b>External</b>`.